validator/client/metrics.go

```diff
@@ -150,6 +161,7 @@ func (v *validator) LogValidatorGainsAndLosses(ctx context.Context, slot uint64)
 		for _, missingPubKey := range resp.MissingValidators {
 			fmtKey := fmt.Sprintf("%#x", missingPubKey)
 			ValidatorBalancesGaugeVec.WithLabelValues(fmtKey).Set(0)
+			ValidatorInclusionDistancesGaugeVec.WithLabelValues(fmtKey).Set(0)
```

Comment:
Why set 0 as default? That is going to be confusing

Reply:
Thanks @terencechain I am not sure if this PR should be included, I do not know if this metric is already exported as derived from others. I set this as draft cause someone in r/ethstaker was asking for something like this. Will leave as no-value perhaps and then each user can decide what to do in Grafana